tutorials/notebooks/example_keras_effdet_lite0.ipynb

```diff
@@ -81,7 +80,8 @@
   {
    "cell_type": "markdown",
    "source": [
-    "In order to convert the PyTorch model, you'll need to use the conversion code in the [MCT tutorials folder](https://github.com/sony/model_optimization/tree/main/tutorials), so we'll clone the MCT repository to a local folder and only use that code. The installed MCT package will be used for quantization. "
+    "In order to convert the PyTorch model, you'll need to use the conversion code in the [MCT tutorials folder](https://github.com/sony/model_optimization/tree/main/tutorials), so we'll clone the MCT repository to a local folder and only use that code. The installed MCT package will be used for quantization. \n",
```

Comment:
not sure "latest version" is correct. maybe just say explicitly that you're running on the latest version of the source code